shiny/http_server.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import copy
+
+import flask
+from google.protobuf import empty_pb2
+from google.protobuf import json_format
+
+from shiny import shiny_pb2
+
+
+app = flask.Flask(__name__)
+
+
+@app.route('/v1/do-nothing/<unicorn_name>', methods=['POST'])
+def do_nothing(unicorn_name):
+    combined = copy.deepcopy(flask.request.json)
+    combined.update(flask.request.args.to_dict())
```

Comment:
@jonparrott It's worth noting here that `flask.request.args` is a multidict, so calling `to_dict()` may clobber some values, e.g.

```python
>>> from werkzeug.datastructures import ImmutableMultiDict
>>> d = ImmutableMultiDict([
...     ('a', 'b'),
...     ('c', 'd'),
...     ('e', 'f'),
...     ('a', 'pickle'),
...     ('c', 'soup'),
... ])
>>> d.to_dict()
{'a': 'b', 'c': 'd', 'e': 'f'}
```